src/Copy/BrandCopy.elm

```diff
@@ -279,40 +279,40 @@ brandCopy key =
 
         -- Supporters Page
         SupportersTitleH2 ->
-            CopyText "✨"
+            CopyText "Our Supporters"
 
         SupportersP1 ->
-            CopyText "✨"
+            CopyText "We would like to thank our funders, whose generosity has made this website possible."
 
         SupportersP2 ->
-            CopyText "✨"
+            CopyText "This web app was funded by CAST - Centre for Acceleration of Social Technology - as part of a project called Contemplating Action which is seeking to find out whether visual stories can help people overcome their fears and barriers to getting support. The project has produced an app that could be used by different charities to encourage people to reach out and get help by reassuring them that they are not alone."
 
         CastLogoAlt ->
-            CopyText "✨"
+            CopyText "Centre for Acceleration of Social Technology logo"
 
         ComicReliefLogoAlt ->
-            CopyText "✨"
+            CopyText "Comic Relief logo"
 
         DepartmentForCultureLogoAlt ->
-            CopyText "✨"
+            CopyText "Department for Culture Media & Sport logo"
 
         BigLotteryLogoAlt ->
-            CopyText "✨"
+            CopyText "Big Lottery logo"
 
         SupportersP3 ->
-            CopyText "✨"
+            CopyText "The initial You Matter app which inspired Contemplating Action was part of Tech vs Abuse. This grant was funded through a joint funding initiative between the Tampon Tax Fund, a partnership between HM Government and Comic Relief, and the Big Lottery Fund."
 
         SupportersP4 ->
-            CopyText "✨"
+            CopyText "The original You Matter web app was developed by Neontribe and The Haven, together with women who have been victims of domestic abuse. The Haven is a Wolverhampton based charity which supports women and dependent children who are vulnerable to domestic violence, homelessness and abuse."
 
         NeontribeLogoAlt ->
-            CopyText "✨"
+            CopyText "Neontribe logo"
 
         HavenLogoAlt ->
-            CopyText "✨"
+            CopyText "The Haven Wolverhampton logo"
 
         SupportersP5 ->
-            CopyText "✨"
+            CopyText "This web app was built by Neontribe and seAp, together with people who have used seAp’s health care complaints service."
 
         FooterSupportersText ->
```

Comment:
We should put the defaults back in here. Might as well with this PR.

Reply:
@katjam added. can we merge?

Reply:
@AlinChindea Go for it.